lib/core/src/display/rendering/html/html_renderer.rs

```diff
@@ -140,28 +156,35 @@ impl HTMLRenderer {
 
     /// Renders the `Scene` from `Camera`'s point of view.
     pub fn render(&self, camera: &mut Camera, scene: &Scene<HTMLObject>) {
-        let trans_cam    = camera.transform.to_homogeneous().try_inverse();
+        let trans_cam    = camera.transform().to_homogeneous().try_inverse();
         let trans_cam    = trans_cam.expect("Camera's matrix is not invertible.");
         let trans_cam    = trans_cam.map(eps);
         let trans_cam    = invert_y(trans_cam);
 
-        // Note [znear from projection matrix]
         let half_dim     = self.renderer.container.dimensions() / 2.0;
         let y_scale      = camera.get_y_scale();
-        let near         = y_scale * half_dim.y;
-
-        js::setup_perspective(&self.data.div, &near.into());
-        setup_camera_transform(
-            &self.data.camera,
-            near,
-            half_dim.x,
-            half_dim.y,
-            &trans_cam
-        );
+        let y_scale      = y_scale * half_dim.y;
+
+        match camera.camera_type() {
+            CameraType::Perspective(_) => {
+                js::setup_perspective(&self.data.div, &y_scale.into());
+                setup_camera_perspective(
+                    &self.data.camera,
+                    y_scale,
+                    half_dim.x,
+                    half_dim.y,
+                    &trans_cam
+                );
+            },
+            CameraType::Orthographic(_) => {
+                setup_camera_orthographic(
+                    &self.data.camera, &trans_cam);
+            }
```

Comment:
These two codes are formatted differently.

Reply:
```rust
                setup_camera_perspective(
                    &self.data.camera,
                    y_scale,
                    half_dim.x,
                    half_dim.y,
                    &trans_cam
                );
``` won't fit the 80 chars limit.

And I don't know why I left it multiline.
```rust
                setup_camera_orthographic(
                    &self.data.camera, &trans_cam);
```

I fixed it to
```rust
setup_camera_orthographic(&self.data.camera, &trans_cam);
```